frontend/tests/utils/xgeneUtils.ts

```diff
@@ -0,0 +1,113 @@
+import { ROUTES } from "src/common/constants/routes";
```

Comment:
Can we rename the file to wmgUtils.ts? Not sure what xgene stands for :laughing: Thank you!